src/components/PageSettingsModal/PageSettingsSchema.jsx

```diff
@@ -36,9 +36,8 @@ export const PageSettingsSchema = (existingTitlesArray = []) =>
     permalink: Yup.string().when("layout", (layout, schema) => {
       switch (layout) {
         case "file":
-          return schema
         case "link":
-          return schema.required("Permalink is required")
+          return schema
```

Comment:
It was removed because we no longer use `Permalink` for `link` type resources - the param that stores the url being linked to is now `external`! So there's no longer any specific schema to check against for this case